assets/diagrams/TECS_throttle.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
What is this file for? It isn't imported or linked, and generally HTML files should not be used directly, and certainly not as assets in diagrams dir.